src/frontend/src/contexts/LanguageContext.tsx

```diff
@@ -1,13 +1,76 @@
 import { i18n } from '@lingui/core';
+import { t } from '@lingui/macro';
 import { I18nProvider } from '@lingui/react';
 import { useEffect } from 'react';
 
 import { api } from '../App';
 import { useLocalState } from '../states/LocalState';
 
 // Definitions
-export type Locales = 'en' | 'de' | 'hu' | 'pseudo-LOCALE';
-export const languages: Locales[] = ['en', 'de', 'hu'];
+export type Locales =
```

Comment:
Why not use keyof languages instead of redecorating them at several different places? If a language is added now, we need to carefully search everywhere where we need to add the new language in be and fe at several places

Reply:
I think I am not experienced enough in ts to understand what you mean. Could you provide a sample of how that would look?

Reply:
Something like 
```ts
export type Locales = keyof languages
```
Typescript [documentation](https://www.typescriptlang.org/docs/handbook/2/keyof-types.html)

Reply:
But this would require, that you remove the typing from the languages obj, so the object type is interfered

Reply:
Implemented, thx for the suggestion